static_src/stores/org_store.js

```diff
@@ -25,22 +25,24 @@ class OrgStore extends BaseStore {
       case orgActionTypes.ORG_FETCH: {
         cfApi.fetchOrg(action.orgGuid);
         this.fetching = true;
+        this.fetched = false;
```

Comment:
We don't need to, but perhaps we should? I feel like emitting changes should be considered a cheap operation and will just cause React to determine if diffs should be flushed to the DOM and act accordingly.

In this case, I think we should emit a change event. When we make a request, we're changing the state of the store and I imagine all views that subscribe to the store care about this particular value change since it likely means they will show a `<Loading /`> component.